interfaces/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "fuel-client-interfaces"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+repository = "https://github.com/FuelLabs/fuel-core"
+description = "Fuel core interfaces"
+
+[dependencies]
+anyhow = "1.0"
+async-trait = "0.1"
+fuel-asm = "0.1"
+fuel-storage = "0.1"
+fuel-tx = { version = "0.2", features = ["serde-types"] }
+fuel-types = { version = "0.1", features = ["serde-types"] }
+fuel-vm = "0.2"
+futures = "0.3"
+hashbrown = "0.12"
```

Comment:
According to the docs:
> Since Rust 1.36, this is now the HashMap implementation for the Rust standard library.

So we probably don't need to manually include this from crates.io anymore?

Reply:
Removed